packages/botonic-plugin-hubtype-analytics/src/index.ts

```diff
@@ -0,0 +1,43 @@
+import { BotRequest, Plugin } from '@botonic/core'
+import axios from 'axios'
+
+import { HtEvent } from './types'
+
+export interface HubtypeAnalyticsOptions {
+  baseUrl: string
+}
+
+export default class BotonicPluginHubtypeAnalytics implements Plugin {
+  baseUrl: string
+  constructor(options: HubtypeAnalyticsOptions) {
+    this.baseUrl = options.baseUrl
+  }
+
+  pre(): void {
+    return
+  }
+
+  post(): void {
```

Comment:
pre/post are no longer needed, I think we left them as optional types

Reply:
Neither is necessary but if there is neither I am not implementing anything from Plguin